src/gpio.rs

```diff
@@ -64,6 +64,10 @@
 use core::marker::PhantomData;
 
 use crate::rcc::ResetEnable;
+// EB: Todo: If making PR upstream, this should only be include for some boards (see pwr.rs)
+use crate::stm32::SYSCFG;
+// EB: Todo: If making PR upstream, check which boards has relevant Pxy_C pins and update
```

Comment:
Done?

Reply:
No, this involves going through the data sheets for all the supported MCUs (there are at least 13) and the package information for each of those (probably 3-5 for each MCU) to see which have these pins (and then add feature flags in the right places), so it is a rather big undertaking...